packages/gatsby/src/schema/__tests__/connection-filter-on-linked-nodes.js

```diff
@@ -3,6 +3,7 @@ const { build } = require(`..`)
 const withResolverContext = require(`../context`)
 const { store } = require(`../../redux`)
 const { actions } = require(`../../redux/actions`)
+import { isLmdbStore } from "../../datastore"
```

Comment:
Is this import not actually used?

Reply:
Not anymore. In the original branch, I was relying on it vs `GATSBY_EXPERIMENTAL_LMDB_INDEXES` env var to conditionally change test expectations. Will remove.